tests/rptest/services/redpanda_installer.py

```diff
@@ -258,8 +258,8 @@ def _setup_head_roots_unlocked(self):
             cmd = None
             if self._head_backed_up:
                 cmd = f"mv /opt/redpanda {head_root_path}"
-            elif not node.account.exists(head_root_path):
-                cmd = f"ln -s {rp_install_path_root} {head_root_path}"
+            else:
+                cmd = f"ln -sf {rp_install_path_root} {head_root_path}"
```

Comment:
@andrwng looks ok?

Reply:
Yeah looks fine, I think, especially given the above comment. I suspect this was originally intended as a micro-optimization to avoid a needless `ln`, but I don't think that is of any concern